infra/core/gateway/api-management.bicep

```diff
@@ -32,21 +32,24 @@ param applicationInsightsName string
 @description('Azure Cache for Redis Service Name')
 param redisCacheServiceName string = ''
 
-resource apimService 'Microsoft.ApiManagement/service@2022-08-01' = {
+var redisConnectionString = !empty(redisCacheServiceName) ? '${redisCache.properties.hostName},password=${redisCache.listKeys().primaryKey},ssl=True,abortConnect=False' : ''
+var redisHostName = !empty(redisCacheServiceName) ? '${redisCache.properties.hostName}' : ''
+
+resource apimService 'Microsoft.ApiManagement/service@2023-03-01-preview' = {
   name: name
   location: location
   tags: tags
   sku: {
     name: sku
-    capacity: (sku == 'Consumption') ? 0 : ((sku == 'Developer') ? 1 : skuCount)
+    capacity: (sku == 'Consumption') ? 0 : ((skuCount == 0) ? 1 : skuCount)
```

Comment:
Developer fails if skuCount != 1 - need to revert this so that it works with sku = 'Developer' and skuCount > 1

Reply:
Done